Lib9c/Action/ClaimMonsterCollectionReward.cs

```diff
@@ -14,25 +14,24 @@
 namespace Nekoyume.Action
 {
     [Serializable]
-    [ActionType("claim_monster_collection_reward")]
+    [ActionType("claim_monster_collection_reward2")]
     public class ClaimMonsterCollectionReward : GameAction
     {
         public Address avatarAddress;
-        public int collectionRound;
         public override IAccountStateDelta Execute(IActionContext context)
         {
             IAccountStateDelta states = context.PreviousStates;
-            Address collectionAddress = MonsterCollectionState.DeriveAddress(context.Signer, collectionRound);
-
+            Address collectionAddress = MonsterCollectionState.DeriveAddress(context.Signer);
+            Address inventoryAddress = avatarAddress.Derive(LegacyInventoryKey);
```

Comment:
it must be add `worldInformation address, questList address` too. because this action might be the first time called `AvatarState.SerializeV2()`.

Reply:
- QuestList can be fair enough, but WorldInformation doesn't seem to relate to this action.
- As you said, we should care about first state migration, but this code will remain and produces meaningless diffs after migration. is there any way to check the avatar state already migrated?